### DISCUSSION TITLE
NFT: Separate Thumbnail from Payload ( different from locked content )

### DISCUSSION THREAD

**Discussion by oO** [deso-protocol/dips#184] [Medium CIPs]
As NFTs move away from simple image formats into video / audio / 3D / ... it's becoming important to have a standard way to represent NFTs as a thumbnail, while also supporting payloads that are not easily shown in a feed or gallery.

Right now creators are using the unlockable content which has a number of issues:

1. the content can only be added at the time of the sale, not before, which is a problem for automated sale.
2. the content can't be added to NFTs with multiple editions.
3. the content may not survive a resale intact.

Proposal to clearly separate an NFT thumbnail ( the current image field ) from an NFT payload ( an additional field, similar to current links to ARweave content ) as well as a standard Content-Type field ( to support automated playback/display of the content.

**Comment by KrassensteinBitClout** [comment]
Great point!  Love this idea.

**Comment by BryanHarrisScripts** [comment]
As mentioned prior ...

Make them tradable cards like hockey cards were in the '70s or pokemon in 2000.

desocialworld.com/posts/b88ceceabf26082164b7e4167dbc1d8b97a09edade2936b4798a713bd2bd564b?feedTab=Language-Feed

https://netikitt.io/netikitt-home-personal-web-page/

**Comment by stet** [comment]
agree but can use PostExtraData with an NFT k/v schema standard.
